pm2-config.json

```
@@ -6,14 +6,14 @@
       "WIR_ENV": "staging",
       "NODE_ENV": "production",
     },
-    "args": "run start:prod -- --backend_url=http://localhost:8080 --port=3000"
+    "args": "run start:prod:remote -- --backend_url=http://localhost:8080 --port=3000"
```

"script": "node", and:
```suggestion
    "args": "server.js --backend_url=http://localhost:8080 --port=3000"
```

---

I don't like the suggestion.
You're hardcoding the script into the config file.
Furthermore - it is repeated in this file.